src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.persistence.*;
 import java.util.Objects;
+import com.google.gson.annotations.SerializedName;
```

Comment:
Added @searializedName(“ ”)
Since this DTO class is being used to connect to a database, I believe it’s a good idea to add @SerializedName annotation. This will make serializing and deserializing much easier to perform. 
Helpful link - https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/SerializedName.html

Reply:
The SerializedNames here are the same as the attribute names themselves... so you should not need SerializedName at all.  SerializedName is best used if there is a mismatch between the database and the attribute names.